proyecto/src/test/java/com/proyecto/proyecto/VacantesTest.java

```diff
@@ -0,0 +1,127 @@
+package com.proyecto.proyecto;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+
+@SpringBootTest
+class VacantesTest {
+
+    @Autowired
+    private RepositorioVacantes repoVacantes;
+
+    @Autowired
+    private RepositorioCategorias repoCategorias;
+
+    @Test
+    void testCrearVacante() {
+        try{
+            SimpleDateFormat sdf = new SimpleDateFormat("dd-MM-yyyy");
+            Vacante vacante = new Vacante();
+            vacante.setId(11);
+            vacante.setNombre("Abogado penalista");
+            vacante.setDescripcion("Abogado con 1 año de experiencia");
+            vacante.setFecha(sdf.parse("10-01-2022"));
+            vacante.setSalario(50000.0);
+            vacante.setEstatus("Creada");
+            vacante.setDestacado(0);
+            vacante.setImagen("logo11.png");
+            vacante.setDetalles("Abogado responsable y proactivo");
+            Categoria categoria = new Categoria();
+            categoria.setId(23);
+            categoria.setNombre("Justicia");
+            categoria.setDescripcion("Abogados de todo tipo");
+            repoCategorias.save(categoria);
+            vacante.setCategoria(categoria);
+            repoVacantes.save(vacante);
+
+            assertNotNull(repoVacantes.findById(11).get());
```

Comment:
#should tendrían que verificarse el resto de los valores establecidos, no sólo que exista el objeto. Hay que verificar la integridad de la información.